sql/moz-fx-data-shared-prod/monitoring_derived/airflow_dag_run_v1/schema.yaml

```diff
@@ -1,36 +1,29 @@
 fields:
```

Comment:
The following schemas need to be updated since they are incompatible with the table that is actually deployed